moveit_experimental/jog_arm/CMakeLists.txt

```diff
@@ -0,0 +1,68 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(jog_arm)
+
+add_compile_options(-std=c++11)
+
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+
+find_package(catkin REQUIRED COMPONENTS
+  roscpp
+  moveit_ros_manipulation
+  moveit_ros_move_group
+  moveit_ros_planning_interface
+  cmake_modules
+  std_msgs
+  tf
+  tf2_geometry_msgs
+)
+
+find_package(Eigen3 REQUIRED)
+
+catkin_package(
+  INCLUDE_DIRS
+    include
+  LIBRARIES
+    compliant_control
+  CATKIN_DEPENDS
+    roscpp
+    moveit_ros_manipulation
+    moveit_ros_move_group
+    moveit_ros_planning_interface
+    tf
+)
+
+include_directories(
+  include
+  ${catkin_INCLUDE_DIRS}
+  ${Eigen_INCLUDE_DIRS}
+)
+
+add_library(compliant_control src/jog_arm/compliant_control/compliant_control.cpp)
+
+add_executable(compliance_test src/jog_arm/compliance_test/compliance_test.cpp)
```

Comment:
add a comment explaining what kind of "test" this is. why not have it with the rostests, below?

Reply:
I'd be happy to write a Python node that publishes fake F/T data for this test, or just move it to a devel branch if you'd rather.